package.json

```diff
@@ -21,12 +21,12 @@
     "preinstall": "npx only-allow pnpm"
   },
   "devDependencies": {
-    "@gadget-client/app-with-no-user-model": "^1.5.0",
-    "@gadget-client/bulk-actions-test": "^1.104.0",
-    "@gadget-client/full-auth": "^1.4.0",
-    "@gadget-client/related-products-example": "^1.859.0",
-    "@gadget-client/zxcv-deeply-nested": "^1.207.0",
-    "@gadget-client/zxcv-simple-relationship": "^1.17.0",
+    "@gadget-client/app-with-no-user-model": "1.6.0-development.5",
+    "@gadget-client/bulk-actions-test": "1.108.0-development.111",
```

Comment:
dev progressed a bit more than the other ones so this one wasn't actually broken but i updated it anyways